cycling-chd-paper/cycling-chd-paper.Rmd

```diff
@@ -73,6 +73,10 @@ active transport at population level in order to inform policy making
 relationship, including socioeconomic status, smoking prevalence, (gender,
 ethnicity and urbanity)
 
+Is the scope just Coronary heart disease? Not cardiovascular disease? Stroke, etc may be interesting in the same vein and may be quite easy to do at the same time
+
+May need to take other confounding variables into account - diet?
```

Comment:
Hi Yann maybe - we can have this discussion via email but suggest in the main text of the paper is not the ideal placee. FYI to add comments use this `<!-- ` symbol like so:
```
<!-- This is a comment -->
```


```
<!--